javascript/validate.js

```diff
@@ -0,0 +1,16 @@
+const formElement = document.querySelector('#form-element');
+const email = formElement.elements.emaile;
+const error = document.querySelector('.error');
+const emailInput = email.value;
+const Emailregex = /^[a-z0-9]+@[a-z0-9-]+\.[a-z0-9-.]+$/;
+const errorMsg = 'Email should only contain lowercase letters.';
+
+formElement.addEventListener('submit', (e) => {
```

Comment:
- [ ] From the requirement when the user submits the form, you check if the email is in lower case and If the validation is OK, the form is sent. In your case, we are getting an error message and the form is not sent. Please fix it to follow the requirements. :+1: 

![Screenshot from 2022-09-09 13-51-12](https://user-images.githubusercontent.com/6615394/189370218-d13aae85-ec23-46f3-8341-899406a45c66.png)